pkg/metrics/delivery_reporter.go

```diff
@@ -155,11 +155,11 @@ func (r *DeliveryReporter) reportEventProcessingTime(ctx context.Context, end ti
 }
 
 func filterTypeValue(v string) string {
-	if v != "" {
-		return v
+	if v == "" {
+		// the default value if the filter attributes are empty.
+		return "any"
```

Comment:
might be worth using a constant. I think we had one somewhere. 
Same for the "custom" one

Reply:
Sure, replaced with constants